space-template.yaml

```diff
@@ -0,0 +1,166 @@
+---
+# These define the potential field types that ca be used in the fields of work
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
+# A map of common strings that can be used to denote states of a work item
+common_work_item_states: &common_work_item_states
+  &state_new new:
+      en_US: "new"
```

Comment:
@michaelvirgil Where do you want to do the translation then? And how? As far as I understood @michaelkleinhenz , we must have i18n by May, right?